client/src/components/ActiveFriends/ActiveFriends.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Avatar, IconButton, Paper, Tooltip, Typography } from '@material-ui/core'
+import PropTypes from 'prop-types'
+import useStyles from './activeFriendsStyles'
+import Preloader from '../Preloader/Preloader'
+import { getAvatarLink } from '../../utils/helpers/imageLinkHelpers'
+import { getActiveTime } from '../../utils/date/getDate'
+import MailOutlineIcon from '@material-ui/icons/MailOutline'
+import { Link } from 'react-router-dom'
+
+const ActiveFriends = ({ activeFriends, activeFriendsAreLoading }) => {
```

Comment:
don't see any usage of activeFriendsAreLoading

Reply:
const loadedContent = activeFriendsAreLoading ? <Preloader /> : friendsList()

Reply:
const loadedContent = activeFriendsAreLoading ? <Preloader /> : friendsList()

Reply:
"const loadedContent = activeFriendsAreLoading ? <Preloader /> : friendsList()"

Reply:
``
  const loadedContent = activeFriendsAreLoading ? <Preloader /> : friendsList()``